packages/desktop-client/src/browser-preload.browser.js

```diff
@@ -133,7 +133,8 @@ document.addEventListener('keydown', e => {
         return;
       }
       e.preventDefault();
-      if (e.shiftKey) {
+      // shift+z
+      if (e.key === 'Z') {
```

Comment:
Is this an oops?

Reply:
I can revert but this was intentional. I think they should behave the same and the old code is more legible in hindsight so I’ll switch back
```suggestion
      if (e.shiftKey) {
```